configure.ac

```diff
@@ -46,5 +46,12 @@ AC_TRY_LINK([
 AC_DEFINE(HAVE_FULL_QUEUE_H,, Define if sys/queue.h includes required macros)
 ])
 
+AC_TRY_LINK([
+	#include <ncursesw/curses.h>
+	#include <ncursesw/term.h>
+], [], [
+AC_DEFINE(HAVE_NCURSESW_H,, Define if ncursesw is available)
+])
```

Comment:
I just realized, won't this be handled by `autoconf`? After building this locally I have a modified `config.h.in` with:

```diff
+/* Define if ncursesw is available */
+#undef HAVE_NCURSESW_H
+
```